eth1/contract_event.go

```diff
@@ -0,0 +1,55 @@
+package eth1
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/bloxapp/ssv/pubsub"
+)
+
+// ContractEvent struct
+type ContractEvent struct {
+	pubsub.BaseSubject
+	Log  types.Log
+	Data interface{}
+}
+
+// Oess struct
+type Oess struct {
+	Index             *big.Int
+	OperatorPublicKey []byte
+	SharedPublicKey   []byte
+	EncryptedKey      []byte
+}
+
+// ValidatorAddedEvent struct
+type ValidatorAddedEvent struct {
```

Comment:
@nivBlox no comments

Reply:
@olegshmuelov pls add your comments

Reply:
done

Reply:
@nivBlox what does the Oess struct name stands for?

Reply:
Ariel decided that name in prd

Reply:
@nivBlox @alonmuroch 
OESS stands for operator encrypted  secret share

Reply:
mm then we need to say that in the comment :)

Reply:
done